starfish/pipeline/features/spots/detector/__init__.py

```diff
@@ -43,6 +51,11 @@ def _cli(cls, args, print_help=False):
 
         print('Detecting Spots ...')
         image_stack = ImageStack.from_path_or_url(args.input)
+        if args.blobs_stack is not None:
```

Comment:
Stacks are loaded in the `cli` code to allow the constructors of the algorithms to require an ImageStack (no strings pointing to filenames).